vendor/vendor-android/src/test/kotlin/com/malinskiy/marathon/android/spek/DeclareMock.kt

```diff
@@ -0,0 +1,27 @@
+package com.malinskiy.marathon.spek
+
+import org.jetbrains.spek.api.dsl.TestBody
+import org.koin.core.context.GlobalContext
+import org.koin.core.definition.BeanDefinition
+import org.koin.core.qualifier.Qualifier
+import org.koin.test.mock.declareMockedDefinition
+import org.koin.test.mock.getDefinition
+
+/**
+ * Declare & Create a mock in Koin container for given type
+ *
+ * @author Arnaud Giuliani
+ */
+inline fun <reified T : Any> TestBody.declareMock(
```

Comment:
Duplicate core/src/test/kotlin/com/malinskiy/marathon/spek/DeclareMock.kt

Reply:
Yeah. Do you want to share this a submodule?

Reply:
Since I've removed koin from these tests I've deleted the android version, so there is only one left in core